content/en/docs/general-usage/cheatsheet.md

```diff
@@ -12,15 +12,40 @@ This cheat sheet gives you an overview of common command line commands to intera
 | | Show device info | `sdv-device-info` |
 | | Device provisioning | `sdv-provision` |
 | | Switch Keyboard layout | `loadkeys de` |
+| System | System load | `htop` |
+| | Disk free | `df -h -t ext4` |
+| | Memory free | `free -h` |
+| Network | Interfaces summary | `networkctl` |
+| | Ethernet status | `networkctl status enp0s2` |
+| | Routing table | `route` |
+| | Active listeners | `netstat -l -n -t` |
 | Kanto-CM | Show all containers| `kanto-cm list` |
 | | User interface | `kantui` |
 | | Service logs | `journalctl -f -l -t container-management` |
-| | Auto deployments | See `/data/containers/manifests/` |
+| | Auto deployments | See `/data/var/containers/manifests/` |
```

Comment:
You can add also `/data/var/containers/manifests_dev/` if you wish to

Reply:
Okay, i will add that as well